cmd/webhook/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM gcr.io/distroless/base:latest
+LABEL maintainers="Kubernetes Authors"
+LABEL description="Snapshot Webhook"
```

Comment:
lets call it VolumeSnapshot, ditto everywhere

Reply:
I'm following the naming convention from the Dockerfile for csi-snapshotter and snapshot-controller. Do you want to change both of those as well, or have an inconsistent naming pattern?